src/controller/python/chip/crypto/p256keypair.py

```diff
@@ -0,0 +1,71 @@
+from typing import *
+from ctypes import *
+
+from ecdsa import NIST256p, SigningKey, ECDH
+import abc
+
+from chip import native
+
+_pychip_ECDSA_sign_msg_func = CFUNCTYPE(c_bool, py_object, c_char_p, c_size_t, c_char_p, POINTER(c_uint32))
+
+_pychip_ECDH_derive_secret_func = CFUNCTYPE(c_bool, py_object, c_char_p, c_char_p, POINTER(c_uint32))
+
+P256_PUBLIC_KEY_LENGTH = 2 * 32 + 1
+
+
+@_pychip_ECDSA_sign_msg_func
+def _pychip_ECDSA_sign_msg(self_: 'P256Keypair', message_buf: c_char_p, message_size: int, signature_buf: c_char_p, signature_buf_size: POINTER(c_uint32)) -> bool:
```

Comment:
Why is it `c_char_p` rather than `c_uint8_p` ?

Reply:
should be c_uint8_p here.